web/src/utils/dropdown.ts

```diff
@@ -0,0 +1,23 @@
+import { Station } from '/Users/conor/projects/wps/web/src/api/stationAPI'
+
+export interface Option {
+  name: string
+  code: number
+}
+
+export const getAutoCompleteOption = (
```

Comment:
Can we get a comment explaining what this does? I imagine looking at this code without context would be confusing to me.

Reply:
Done -- also renamed a bunch of stuff to make this more clear.